templates/default/mesos-init.erb

```diff
@@ -1,6 +1,6 @@
-description "mesos <%= @type %>"
+description "Upstart job for <%= @wrapper %>"
```

Comment:
Would it be safe to say that when we start supporting centos/rhel 7 we will need to add an equivalent systemd init script?